src/sqlancer/clickhouse/ClickHouseProvider.java

```diff
@@ -133,4 +135,9 @@ public Connection createDatabase(ClickHouseGlobalState globalState) throws SQLEx
     public String getDBMSName() {
         return "clickhouse";
     }
+
+    @Override
+    public LoggableFactory getLoggableFactory() {
+        return new SQLLoggableFactory();
```

Comment:
For code reuse, it would be nice to implement a `SQLProviderAdapter` class that extends `ProviderAdapter` and implements the `getLoggableFactory` method, so that this method only needs to be implemented once for all SQL-based DBMSs.